### PR TITLE
Adding changelogs.md to stack modules and Adding ModuleVersion dependency of AzureRm.Profile in Uber modules

### PR DESCRIPTION
**Pull request by bganapa**
## Description

Adding change logs to stack modules

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/StackAdmin/Compute/Commands.Compute/Commands.Compute.csproj [hunk 1]
```diff
@@ -338,9 +338,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <None Include="..\AzureRM.Compute.psd1">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```
Comment (cormacpayne) [inline src/StackAdmin/Compute/Commands.Compute/Commands.Compute.csproj]: @bganapa do we not want to copy this file to the output directory?

Comment (bganapa) [reply src/StackAdmin/Compute/Commands.Compute/Commands.Compute.csproj]: good catch!.. added copy always

**Comment by bganapa** [comment]
ps-sign https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1393/

File: src/StackAdmin/AzureRM/AzureRM.psd1 [hunk 2]
```diff
@@ -12,7 +12,7 @@
     RootModule             = "AzureRM.psm1"
 
     # Version number of this module.
-    ModuleVersion          = '2.3.0'
+    ModuleVersion          = '2.3.0.1'
```
Comment (cormacpayne) [inline src/StackAdmin/AzureRM/AzureRM.psd1]: @bganapa nit: any reason this wasn't a patch version bump (`2.3.1`) like `AzureStack.psd1` down below?

Comment (bganapa) [reply src/StackAdmin/AzureRM/AzureRM.psd1]: No particular reason. I just wanted to bump it to something higher.. This is not a version that we will publish anyway. The version bump is just for nightly build consumption..

Comment (bganapa) [reply src/StackAdmin/AzureRM/AzureRM.psd1]: @cormacpayne pls let me know if you want me to change to 2.3.1

**Pull request merged**